feature-libs/organization/package.json

```diff
@@ -38,8 +38,8 @@
     "@angular/router": "^10.1.0",
     "@angular-devkit/schematics": "^10.1.0",
     "rxjs": "^6.6.0",
-    "@spartacus/core": "3.0.0-next.5",
-    "@spartacus/storefront": "3.0.0-next.5",
+    "@spartacus/core": "3.0.0-next.6",
+    "@spartacus/storefront": "3.0.0-next.6",
     "@spartacus/schematics": "3.0.0-next.5"
```

Comment:
Can you please in different PR fix release-it config for the schematics dependency?